Update System Requirements Doc (#1525)
cyberbotics/webots: docs/guide/system-requirements.md

Webots is provided for Linux 64 (x86-64) systems.
Since Webots 8.1.0, the Linux 32 (i386) version is no longer provided.
Webots doesn't run on Ubuntu versions earlier than 12.04.
- Windows: Webots runs on Windows 10 and Windows 8.1.
It is not supported on older versions (8, 7, Vista or XP) as well as any 32-bit variant.
- Mac: Webots runs on macOS 10.15 "Catalina" and 10.14 "Mojave".

> Webots may work but is not officially supported on earlier versions of macOS, nor below those mentioned.

Reviewer: > on earlier versions of macOS

> nor below those mentioned

Those two sentences means the same no?

Author: To me "earlier" in this context means the versions which might have been just released

Author: Or we could just use above and below...

Reviewer 2: I would simply drop ", not below those mentioned." which is too verbose to me and doesn't really help.

> Webots may work but is not officially supported on earlier versions of macOS, nor below those mentioned.

Reviewer 2: I would make this sentence generic to all OSes:
```suggestion

Webots may work but is not officially supported on earlier versions of the above mentioned operating systems.
```
